packages/expo-cli/src/commands/publish.js

```diff
@@ -27,6 +30,18 @@ export async function action(projectDir: string, options: Options = {}) {
     );
     process.exit(1);
   }
+  const hasOptimized = fs.existsSync(projectDir + '/.expo-shared/assets.json');
```

Comment:
use `path.join` rather than string concatenation here

Reply:
this `.expo-shared` folder will start showing up for people and I wonder if we should add some explicit instructions in the command (maybe only the first time it's run) that people should commit this folder -- what do you think @sergeichestakov ?

Reply:
I pushed a commit addressing that issue 1 min before you made this comment 😄 Let me know what you think about the message in b2a0d49

Reply:
😄 👌 looks good!

Reply:
you read my mind